downloads/swan-lake-release-notes/swan-lake-2201.8.9/RELEASE_NOTE.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: ballerina-left-nav-release-notes
+title: Swan Lake Update 8 (2201.8.9) 
+permalink: /downloads/swan-lake-release-notes/2201.8.9/
+active: 2201.8.9
+---
+
+## Overview of Ballerina Swan Lake Update 8 (2201.8.9)
+
+<em>Swan Lake Update 8 (2201.8.9) is the ninth patch release of Ballerina 2201.8.0 (Swan Lake Update 8) and it includes a new set of bug fixes to the language.</em>
+
+## Update Ballerina
+
+Run the command below to update your current Ballerina installation directly to 2201.8.9 by using the [Ballerina Update Tool](/learn/update-tool/).
+
+```
+$ bal dist pull 2201.8.9
+```
+
+## Install Ballerina
+
+If you have not installed Ballerina, then, download the [installers](/downloads/#swanlake) to install.
+
+## Language updates
+
+### Bug fixes
+
+To view bug fix, see the GitHub milestone for Swan Lake Update 8 (2201.8.9) of the repositories below.
+- [Compiler](https://github.com/ballerina-platform/ballerina-lang/issues/43409)
```

Comment:
Do we need to include the issue link in the release notes?

Reply:
It has been used in previous release notes as well. I can change it if you want to.

Reply:
We have used the milestone in this one https://github.com/ballerina-platform/ballerina-dev-website/blob/9deb541e00e52e3b830f6e60391634284d529037/downloads/swan-lake-release-notes/swan-lake-2201.10.1/RELEASE_NOTE.md?plain=1#L28

@MaryamZi what is the best practice?

Reply:
@MaryamZi In release notes we need to mention the bug fixes in this manner. Right?
https://github.com/ballerina-platform/ballerina-dev-website/blob/9deb541e00e52e3b830f6e60391634284d529037/downloads/swan-lake-release-notes/swan-lake-2201.10.1/RELEASE_NOTE.md?plain=1#L28

Reply:
Generic link as Sarani suggested is the best practice.

Reply:
Sure. Changed